packages/api-client-core/src/InternalModelManager.ts

```diff
@@ -15,7 +15,7 @@ import {
   hydrateRecord,
   hydrateRecordArray,
 } from "./support.js";
-import type { InternalFieldSelection, InternalFindListOptions, InternalFindManyOptions, InternalFindOneOptions } from "./types";
+import type { InternalFieldSelection, InternalFindListOptions, InternalFindManyOptions, InternalFindOneOptions } from "./types.js";
```

Comment:
Feels odd. Curious as to why this one was necessary.

Reply:
All esm files gotta have the extension!

Reply:
I guess the thing I find surprising is that `tsc` wouldn't deal with this, eh? Just given it knows the target.